qubesmanager/qube_manager.py

```diff
@@ -395,6 +394,10 @@ def load_manager_settings(self):
     def get_vms_list(self):
         return [vm for vm in self.qubes_app.domains]
 
+    def update_single_row(self, vm):
+        if vm in self.qubes_app.domains:
```

Comment:
This looks suspicious. What if vm isn't in `self.qubes_app.domains`? Should be removed from the table?
And what if `vm.qid` is not in `self.vms_in_table`?